README.adoc

```diff
@@ -20,33 +20,46 @@ https://github.com/gosecure/malboxes
 
 == Requirements
 
-* Python 3
+* Python 3.3+
+* appdirs
 * jinja2
-* packer (sometimes called packer-io)
-* vagrant
+* packer: https://www.packer.io/intro/getting-started/setup.html
+* vagrant: https://www.vagrantup.com/downloads.html
+
+== Installation
+
+    sudo pip install git+https://github.com/GoSecure/malboxes.git@pip-packaging#egg=malboxes
```

Comment:
Good point for pip3! I use arch so pip is a symlink to pip3 but you are right about most distro being pip -> pip2.